### PR TITLE
Gradient norm clipping should be done right before calling the optimiser

### PR DESCRIPTION
**Pull request by pminervini**
Right now it's done after each step in the gradient accumulation. What do you think?

### CODE REVIEW

**Comment by thomwolf** [comment]
Oh yes, great, thanks, Pasquale. Would you mind fixing the `run_glue` and `run_ner` examples as well?

**Comment by pminervini** [comment]
@thomwolf done! what's the best way to check this code before merging?

**Comment by thomwolf** [comment]
Thanks a lot!
It should be fine, we have continuous integration tests on `run_glue` and `run_squad` so if it passed at least the code run.
